test/runtests.jl

```diff
@@ -60,15 +61,29 @@ macro includetests(testarg...)
     end
 end
 
-gl = global_logger()
-level = get(ENV, "PS_LOG_LEVEL", "Error")
-log_level = get(LOG_LEVELS, level, nothing)
-if log_level == nothing
-    error("Invalid log level $level: Supported levels: $(values(LOG_LEVELS))")
+function get_logging_level(env_name::String, default)
+    level = get(ENV, env_name, default)
+    log_level = get(LOG_LEVELS, level, nothing)
+    if log_level == nothing
+        error("Invalid log level $level: Supported levels: $(values(LOG_LEVELS))")
+    end
+
+    return log_level
 end
-global_logger(ConsoleLogger(gl.stream, log_level))
 
-# Testing Topological components of the schema
-@testset "Begin PowerSystems tests" begin
-    @includetests ARGS
+console_level = get_logging_level("PS_LOG_LEVEL", "Error")
+file_level = get_logging_level("PS_CONSOLE_LOG_LEVEL", "Info")
+
+open_file_logger(LOG_FILE, file_level) do file_logger
+    console_logger = ConsoleLogger(stderr, console_level)
+    multi_logger = MultiLogger([console_logger, file_logger],
+                               LogEventTracker((Logging.Info, Logging.Warn, Logging.Error)))
+    global_logger(multi_logger)
```

Comment:
I would go one step further and create utility functions around this:

```
    console_logger = ConsoleLogger(stderr, console_level)
    multi_logger = MultiLogger([console_logger, file_logger],
                               LogEventTracker((Logging.Info, Logging.Warn, Logging.Error)))
    global_logger(multi_logger)
```

So that the user can call say `create_logger(file=true, console=false)` and use that function in this context. This is not high priority.

Reply:
In the future we need a way to make sure other packages can use a `PowerSystems` logger in conjunction with a logger that they create. I'm not sure how that would work though?

Reply:
I added an exported function called configure_logging.  I ran a test with this function in PowerSimulations - it worked fine.  
logger = PowerSystems.configure_logging(...)

Every log statement is tagged with @<module name> <file:line>.
Getting different loggers (and log files) for different modules is possible but not practical for this PR.